client/src/app/containers/Alert/index.jsx

```diff
@@ -26,7 +26,7 @@ const Alert = () => {
     }
   };
   return (
-    <div className='fixed w-half-screen right-0 px-3 py-2'>
+    <div className='fixed w-half-screen right-0 px-3 py-2 z-50'>
```

Comment:
How would we style this using Material? Can we get rid of the tailwind classes?

Reply:
🤷  it'll be good to move out of tailwind (I think), but I'm not in a _huge_ rush.